### ISSUE TITLE
Comparing two ifc models

### ISSUE THREAD

**Issue by Kirill-basic** [xBimTeam/XbimWebUI#344]
Hello! I've been trying to find any way to properly compare two ifc models, but couldn't find a way to do this. So is it possible to do using xbim? Thanks!

**Comment by martin1cerny** [comment]
Yes, this is possible to do with xbim. You are asking in the WebUI project so, are you interested in the visualization of the comparison, or the actual comparison of models?

**Comment by Kirill-basic** [comment]
I dont actually need to visuilize the result of the comparisson as i've created this question here by mistake. My bad. I need to compare two ifc models and find the difference between them and then save this difference. So this should be some kind of versioning. I know that i can compare the text of this models, but this approach doesn't seem reliable for me. Maybe i'm wrong about it. How can I compare them?
